plugins/package-managers/conan/src/funTest/assets/projects/synthetic/conan-expected-output-py.yml

```diff
@@ -1,36 +1,197 @@
 ---
 project:
-  id: "Conan::Poco:1.9.3"
+  id: "Conan::poco:null"
```

Comment:
We should actually never get `null` for the version of a project. Let me take over your PR and fix this on top of it. Nothing to be done from your side @rettichschnidi.